src/absil/ilwrite.fs

```diff
@@ -4212,7 +4212,7 @@ let writeBinaryAndReportMappings (outfile, ilg, pdbfile: string option, signer:
             else flags
           writeInt32AsUInt16 os dllCharacteristics
        // 000000e0 
```

Comment:
It looks like this was done to make the sections easier to skip through visually.  The whole block of code probably needs some rework outside of this PR.